components/integrations/abstract_outgoing_webhook.tsx

```diff
@@ -168,68 +142,75 @@ export default class AbstractOutgoingWebhook extends React.PureComponent {
         const hook = {
             team_id: this.props.team.id,
             channel_id: this.state.channelId,
-            trigger_words: triggerWords,
-            trigger_when: parseInt(this.state.triggerWhen, 10),
-            callback_urls: callbackUrls,
+            trigger_words: this.state.triggerWords.split('\n'),
```

Comment:
this is changing behavior, `triggerWords` its not from state but from here https://github.com/mattermost/mattermost-webapp/pull/11403/files#diff-1c079508ea259ce1d652c2e8a5fc2539ea8476c81329a976b032cc933006a577L120